metadata.tvshows.thetvdb.com.v4.python/resources/lib/tvdb.py

```diff
@@ -465,12 +465,25 @@ def get_series_episodes_api(self, id, settings):
         return result
 
     def get_episode_details_api(self, id, settings):
-        ep = self.get_episode_extended(id)
-        language = get_language(settings)
         try:
-            trans = self.get_episode_translation(id, language)
-        except requests.HTTPError:
-            trans = self.get_episode_translation(id, "eng")
+            ep = self.get_episode_extended(id)
+        except requests.HTTPError as e:
+            logger.warning(f'No episode found with id={id}. [error: {e}]')
+            return None
+
+        trans = None
+
+        language_attempties = list( dict.fromkeys([get_language(settings), "eng"]) )
```

Comment:
What is the purpose if this weird construction?

Also I'd suggest to adhere to PEP-8 style guide for better code maintainability.

Reply:
Python has `set` data structure since forever, so no need for such elaborate constructions. All can be replaced with:
```python
language_attempties = {get_language(settings), "eng"}
```

Reply:
Thanks @romanvm. Good catch! Applied the fix.

Reply:
Wait wait... 


"Set" will not work in this case specifically, as we need to consider the client option (`get_language(settings)`) always as the primary option, and `"eng"` as the fallback.

As we can't guarantee the order using sets, I just got this "bug" on the test I had written:

```bash
% python test_thetvdb.py TestEpisodesApi.test_get_episode_details_api_successfull_other_lang_translation
testing with 'jpn'
.
----------------------------------------------------------------------
Ran 1 test in 1.496s

OK

% python test_thetvdb.py TestEpisodesApi.test_get_episode_details_api_successfull_other_lang_translation
testing with 'eng'
F
======================================================================
FAIL: test_get_episode_details_api_successfull_other_lang_translation (__main__.TestEpisodesApi)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "[...]/metadata.tvshows.thetvdb.com.v4.python/test_thetvdb.py", line 36, in test_get_episode_details_api_successfull_other_lang_translation
    self.assertEqual('完結〜私の最終決断!!父娘愛が起こす奇跡', ep['name'], 'Episode title does not match.')
AssertionError: '完結〜私の最終決断!!父娘愛が起こす奇跡' != 'Episode 10'
- 完結〜私の最終決断!!父娘愛が起こす奇跡
+ Episode 10
 : Episode title does not match.

----------------------------------------------------------------------
Ran 1 test in 1.279s

FAILED (failures=1)
```

First run, `{get_language(settings), "eng"}` was equivalent to `{"jpn", "eng"}`.
Second run, it failed because the set was ordered this way: `{"eng", "jpn"}`

Reply:
FYI, reverted the set in https://github.com/thetvdb/metadata.tvshows.thetvdb.com.v4.python/pull/12/commits/90d18cf6099ccb5eae40ef8a2392a16732dcc87e#diff-d79cbad48c067b7498825df4477eb3b6fe7a9fe39b84668f41fce3280587c811R479-R480